docs/source/contribution/developer_contributor_guidelines.md

```diff
@@ -116,19 +116,19 @@ We will work with you to complete your contribution, but we reserve the right to
 
 ## Create a pull request
 
-Create your pull request with [a descriptive title](#pull-request-title-conventions). Before you submit it, consider the following:
+[Give your pull request a descriptive title](#pull-request-title-conventions). Before you submit it, consider the following:
 
 * You should aim for cross-platform compatibility on Windows, macOS and Linux
 * We use [SemVer](https://semver.org/) for versioning
```

Comment:
```suggestion
* We use [Semantic Versioning](https://semver.org/) for versioning
```
When I was borrowing from these contribution guidelines for internal use, I got a recommendation to spell out Semantic Versioning; I think it makes sense?